src/CommunityToolkit.Maui/Alerts/Snackbar/Snackbar.ios.macos.cs

```diff
@@ -77,7 +75,7 @@ static CGRect GetCornerRadius(CornerRadius cornerRadius)
 		return new CGRect(cornerRadius.BottomLeft, cornerRadius.TopLeft, cornerRadius.TopRight, cornerRadius.BottomRight);
 	}
 
-	sealed class AppleSnackbar : NativeToast, IDisposable
+	sealed class AppleSnackbar : Views.Toast.Toast, IDisposable
```

Comment:
let's rename it to AppleToast

Reply:
My preference to keep it as `Toast` in case we decide to add a `Toast` control to Maui Community Toolkit in the future. 

Then, similar to the Snackbar architecture, we'd have `Toast.shared.cs` `Toast.ios.macos.cs`, `Toast.android.cs`, and `Toast.windows.cs`.

The filename is already `Toast.ios.macos.cs` which means it can only run on iOS + MacCatalyst anyways.